specification/migration/get_feature_upgrade_status/GetFeatureUpgradeStatusResponse.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Indices, VersionString } from '@_types/common'
+
+export class Response {
+  body: {
+    features: MigrationFeature[]
+    migration_status: string
+  }
+}
+
+export class MigrationFeature {
+  feature_name: string
+  minimum_index_version: VersionString
+  migration_status: string
+  indices: Indices
```

Comment:
From this XContent builder https://github.com/elastic/elasticsearch/blob/26c86871fc091900952e88e252c36fbfedf8d5fa/server/src/main/java/org/elasticsearch/action/admin/cluster/migration/GetFeatureUpgradeStatusResponse.java#L298 I think the structure for `indices` is:

```ts
export class MigrationFeatureIndexInfo {
  index: IndexName
  version: VersionString
  failure_cause?: ErrorCause
}
```

and then `MigrationFeature.indices` can be `MigrationFeatureIndexInfo[]`.

Reply:
Thank you for elaborating, addressed the feedback in https://github.com/elastic/elasticsearch-specification/pull/1393/commits/794f0f4f2b0903dea03ebee6f794163aad6c46bf